pkg/alerts/alertsHandler/alertsHandler.go

```diff
@@ -298,7 +298,13 @@ func ProcessTestContactPointRequest(ctx *fasthttp.RequestCtx) {
 }
 
 func testWebhookURL(webhookURL string) error {
-	resp, err := http.Get(webhookURL)
+	client := alertutils.GetCertErrorForgivingHttpClient()
+	req, err := http.NewRequest("GET", webhookURL, nil)
+	if err != nil {
+		log.Errorf("testWebhookURL: failed to create a test webhook request - %v", err)
```

Comment:
nit: typically we use the format 
```
log.Errorf("....., request, err: %v", err)
```
two things
1) prefix the var name, for e.g. : err 
2) use a colon or equals seperator after the var name instead of a dash, just to keep it uniform across our entire code base. One could argue dash is also fine, but then everywhere we should just use the same separator